### ISSUE TITLE
Unable to drap and drop using firefox for mobile.

### ISSUE THREAD

**Issue by juanjosepablos** [FloatingOctothorpe/python-kanban#9]
Hi,
Thank you for sharing this.I was testing your software,and I found that I could not drag and drop items using Firefox for android. 126.0.1.
I had upgrade to vue.min.js Latest stable version: 2.7.16 but still not luck.
Test same Firefox version on Ubuntu 20.04 and it works just fine.

**Comment by FloatingOctothorpe** [comment]
Hi, thanks for pointing that out.

I can replicate the same behaviour. The JavaScript which handles dragging/dropping tasks currently uses the [dragstart](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragstart_event) event which I suspect isn't firing on touch based devices. I'll have a go at implementing the relevant touch events when I get some time.

**Comment by juanjosepablos** [comment]
I was doing a bit of research and adding this line works for me:

```
diff --git a/static/index.html b/static/index.html
index a1c8d38..71fc7c6 100644
--- a/static/index.html
+++ b/static/index.html
@@ -194,6 +194,7 @@
       transition: transform 0.3s;
     }
   </style>
+    <script id="DragDropTouch" src="https://bernardo-castilho.github.io/DragDropTouch/DragDropTouch.js"></script>
 </head>
 <body>
   <noscript>

```
get more info here:
https://github.com/Bernardo-Castilho/dragdroptouch

**Comment by FloatingOctothorpe** [comment]
Resolved by #10

**Comment by juanjosepablos** [comment]
> Resolved by #10

not for me...
